requirements.txt

```diff
@@ -1,2 +1,2 @@
-paho-mqtt==1.1
-robotframework==2.8.7
+paho-mqtt>=1.1
+robotframework>=3.0
```

Comment:
same:
```suggestion
robotframework~=3.1
```

Reply:
Please see comment for `paho-mqtt` above.